### DISCUSSION TITLE
Import SecureCRT backup/transfer XML into sessions in WebSSH

### DISCUSSION THREAD

**Discussion by Z1mDMan** [isontheline/pro.webssh.net#1005] [Ideas]
I use SecureCRT for work and would like to be able to do an import of my sessions into WebSSH from a backup of the SecureCRT sessions XML file. Is that a possibility for a feature?

**Comment by isontheline** [comment]
Hello @Z1mDMan 👋

Many thanks for your interest in WebSSH 🙏

I'm merging your request to this issue : https://github.com/isontheline/pro.webssh.net/issues/252

**Reply by isontheline** [reply]
I will work on it soon

**Reply by isontheline** [reply]
@Z1mDMan 

1. Do you have a lot of servers?
2. Do you need only servers to be imported? Or credentials too?
